frontend/components/pages/volunteer/faq.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from "react";
+import { IFaq } from "types/faq";
+
+export const FAQS: React.FC<{
+  faq: IFaq;
+}> = ({ faq }) => {
+  return (
+    <>
+      <div className="dropdown">
```

Comment:
suggestion - I know we are not at the point of really styling these components yet, but in the short term it might be nice to use the native HTML details element just to provide a convenient way to click a question and show answers. Can just keep it super simple like:
```tsx
<details>
    <summary>{faq.Question}</summary>
    {faq.Response}
</details>
```
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details

![image](https://user-images.githubusercontent.com/10515065/184003090-cf55f91e-79a7-45d3-8f03-b30a424c6bca.png)

Reply:
Let me get to work @chrismclarke